API/Startup.cs

```diff
@@ -21,15 +24,27 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<Iserializer, JsonSerializer>();
             services.AddSingleton<IErrorFormatter, JsonErrorFormatter>();
+            //services.AddDbContext<ApiContext>(opt => opt.UseInMemoryDatabase("API"));
+            var builder = new MySqlConnectionStringBuilder(Configuration.GetConnectionString("DefaultConnection"));
+            builder.Server = Configuration["Server"];
+            builder.Database = Configuration["Database"];
+            builder.UserID = Configuration["Uid"];
+            builder.Password = Configuration["DbPassword"];
```

Comment:
It is possible to use an initializer like this:
```
var builder = new MySqlConnectionStringBuilder(Configuration.GetConnectionString("DefaultConnection"))
{
    Server = Configuration["Server"],
    Database = Configuration["Database"],
    UserID = Configuration["Uid"],
    Password = Configuration["DbPassword"]
};
```
Maybe ot would be better to change it like this in order to ensure object is created without any errors or missed properties?

Reply:
Yes, thank you for noticing.